test/percy/README.md

```diff
@@ -4,6 +4,105 @@ We use visual regression testing to avoid inadvertently introducing visual regre
 
 This works by taking screenshots of our components in different states. When changes are made, they get compared to the baseline on CI by [Percy](https://percy.io/). In case changes are detected, the CI check will fail unless somebody approves the changes on the Percy website.
 
+- [Scripts overview](#scripts-overview)
+  - [Running on CI](#running-on-ci)
+  - [Running tests locally to debug components](#running-tests-locally-to-debug-components)
+  - [Running tests locally to debug snapshots](#running-tests-locally-to-debug-snapshots)
+  - [In detail](#in-detail)
+    - [yarn build](#yarn-build)
+    - [yarn visual-testing-app:build](#yarn-visual-testing-appbuild)
+    - [yarn visual-testing-app:serve](#yarn-visual-testing-appserve)
+    - [yarn percy](#yarn-percy)
+- [Constraints](#constraints)
+  - [How to write visual regressions tests](#how-to-write-visual-regressions-tests)
+    - [Create files ending in `.visualroute.js`](#create-files-ending-in-visualroutejs)
+    - [Write tests in ending `.visualspec.js`](#write-tests-in-ending-visualspecjs)
+  - [Why we test the bundle](#why-we-test-the-bundle)
+  - [Why we group multiple states into one snapshot](#why-we-group-multiple-states-into-one-snapshot)
+- [Adding Visual Regression Routes](#adding-visual-regression-routes)
+- [Adding Visual Regression Tests](#adding-visual-regression-tests)
+  - [Suite](#suite)
+  - [Spec](#spec)
+
+## Scripts overview
+
+### Running on CI
+
+```bash
+# build ui-kit first
+yarn build
+
+# now build the visual-testing-app
+yarn visual-testing-app:build
+
+# run percy tests
+yarn percy --reporters jest-silent-reporter
+```
+
+### Running tests locally to debug components
+
+```bash
+# build ui-kit in watch mode
+yarn build:watch
+
+# now build the visual-testing-app
+yarn visual-testing-app:build
+
+# serve the visual-testing-app
+# you can now visit http://localhost:3001 to see what the components look like
+yarn visual-testing-app:serve
+```
```

Comment:
none of this runs the tests... it just serves the visual-testing-app. Need to rename the title

Reply:
Done 👍